app/views/layouts/provider.html.slim

```diff
@@ -18,7 +18,7 @@ html[lang="en" class="pf-m-redhat-font"]
       = render partial: '/shared/provider/header'
       = render partial: 'shared/provider/navigation/vertical_nav' unless active_menu == :dashboard
       #border.pf-c-page__main class="self_clear" role="main"
-        = render 'shared/provider/flash', flash: flash
+        = render 'shared/provider/flash', flash: flash unless flash.empty?
```

Comment:
This is dangerous because then if in the 1st page it visits with this layout there isn't a flash and then it goes to another page with the same layout (or even staying in the same page but doing an ajax request) that contains a flash, it will not appear although it should.

Reply:
yes, like commented https://github.com/3scale/porta/pull/1629#issuecomment-582312908